client/src/app/core/repositories/assignments/assignment-repository.service.ts

```diff
@@ -210,7 +210,8 @@ export class AssignmentRepositoryService extends BaseAgendaContentObjectReposito
      * @param assignment
      */
     public async sortCandidates(sortedCandidates: any[], assignment: ViewAssignment): Promise<void> {
-        throw Error('TODO');
+        throw new Error('TODO');
+        // TODO responds with 405 method not allowed:
         // const restPath = `/rest/assignments/assignment/${assignment.id}/sort_related_users`;
         // const data = { related_users: sortedCandidates };
         // await this.httpService.post(restPath, data);
```

Comment:
@FinnStutzenstein The server does not seem to like this request. Could you please take a look?